### ISSUE TITLE
(wontfix) Endless Loop with GH Auth on Sidekiq

### ISSUE THREAD

**Issue by ewilson-adhoc** [department-of-veterans-affairs/va.gov-team#40248]
## Expected Behavior
GH Auth should redirect to the application after the user has authenticated. 

## Actual Behavior
GH Auth should redirect to the application after the user has authenticated. 

## To Reproduce
I switched sidekiq UI on development to use basic auth for the short-term to unblock other work. The manifest needs to be updated to use "develop" instead to switch the application back to using GH Auth https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/platform-pst-iet/vets-api-sidekiq-ui/dev

Note -- there haven't been any code changes in the last month. It's unclear why this suddenly stopped working, but it's probably something related to configurations.

**Comment by ewilson-adhoc** [comment]
After conversations with AMT - we may move away from using GH auth as our authN provider in favor of something like Keycloak -- this bug should be closed WONTFIX and we should pull #40061 into the sprint, or we should modify this story to reflec the work that is being done.

**Comment by ewilson-adhoc** [comment]
Closing wontfix -- will address this problem as part of https://app.zenhub.com/workspaces/integration-experience---platform-spike-team-61f18a64735e8e0015911cf7/issues/department-of-veterans-affairs/va.gov-team/41509
